tutoraspects/templates/aspects/apps/superset/pythonpath/queries/dim_course_videos.sql

```diff
@@ -27,4 +27,5 @@ select
 from
     courses
     join video_blocks
-        using (course_key)
+        on (courses.org = video_blocks.org
```

Comment:
I don't have a clean reproduction of the issue I was seeing, but I ran into cases where joins that used the `using` syntax were failing with "column not found" errors. I'll see if I can put together a ClickHouse fiddle with the behavior but in the meantime I wanted to get these changes in.

Reply:
With the instructor dashboard, I've experienced weird cases with the demo data. Some behaviors:

- Dashboard is blank, with just filters and no charts, just a loading screen.
- Some charts randomly fail. Needs to force refresh to appear
- As the org filter is required, and the filters are cached, no organization appears in the selector. Not sure if it's a bug with the demo data or superset cache.

Reply:
Does it seem like this is happening only when you merge new changes, or does it happen even if you just look at the dashboard without changing anything?

Since I'm also working with the demo course data, my guess is that it's related to the Superset cache. I've noticed some odd behavior after importing assets. Hopefully this is just related to refreshing the assets and occurs less frequently once the changes slow down.